### DISCUSSION TITLE
Import M3U Playlists

### DISCUSSION THREAD

**Discussion by bsammon** [VinylMusicPlayer/VinylMusicPlayer#935] [Q&A]
Can Vinyl Music Player read a M3U playlist created by another app (a text editor)?

I'm not sure if this is an issue of me not knowing the right thing to do, or if the feature is not there.

Maybe VMP is picky about the format of the M3U file?

**Comment by soncaokim** [comment]
Vinyl does not directly read playlist files as of now, but Android
underlying media system does (at its own rhythm, not always up to date) and
it exposes those files as playlists to Vinyl.

Once a new playlist is picked up by Android, Vinyl will have the new one
displayed.

--

On the format, I'd suggest you to pick any playlist in vinyl, do a "Save as
file" and have a look at the generated .m3u file. I've based on that in the
past, and Android interpreted the manually crafted .m3u files correctly.

--


For changes on existing playlists, there is an improvement needed on Vinyl
side to pickup the changes.
